### PR TITLE
:robot: Copy sarif files from proper folder

### PR DESCRIPTION
**Pull request by Itxaka**
**What this PR does / why we need it**:
Fixes the sarif dir under release workflow

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Partially fixes https://github.com/kairos-io/kairos/issues/1200

### CODE REVIEW

**Review by mauromorales — APPROVED** [review]
LGTM

File: .github/workflows/release-arm.yaml [hunk 1]
```diff
@@ -102,7 +102,7 @@ jobs:
           MODEL: ${{ matrix.model }}
         run: |
           mkdir sarif
-          sudo mv release/*.sarif sarif/
+          sudo mv build/*.sarif sarif/
```
Comment (jimmykarily) [inline .github/workflows/release-arm.yaml:105]: doesn't `release.yaml` also need the same fix?

Comment (Itxaka) [reply .github/workflows/release-arm.yaml:105]: weirdly enough no, it seems to work in that job: https://github.com/kairos-io/kairos/actions/runs/4532595843/jobs/8008458849

Comment (Itxaka) [reply .github/workflows/release-arm.yaml:105]: oh I see, the release job moves the files from /build to /release....

Comment (Itxaka) [reply .github/workflows/release-arm.yaml:105]: not sure why on arm we dont directly do that or why we move the files into release/ and not work with the build directory directly. Probably due to permission stuff?

**Pull request merged**